starter-code/lib/viking.js

```diff
@@ -19,13 +19,82 @@
 //   FootViking: FootViking
```

Comment:
You should keep this assignment to reference back to later.  Hope you understand what you did here b/c it's a good example of prototypal inheritance.